### DISCUSSION TITLE
Using a PrimeVue template in Nuxt

### DISCUSSION THREAD

**Discussion by Alparslan524** [primefaces/community#1988] [PrimeVue Templates] [answered]
I purchased a Diamond template in PrimeVue, but I want to develop a project in Nuxt and not in Vue. Is there any way to download this Diamond template for Nuxt or integrate it into a Nuxt project?
Or is there a ready-made project for Nuxt and not for Vue?

**Comment by tugcekucukoglu** [comment, marked as answer]
Sakai-Nuxt project could be a very-first starter. Also, Apollo has Nuxt support.

**Reply by janderegg** [reply]
Sakai-Nuxt is no more on github? I find only the "Vite-Version"...
https://github.com/primefaces/sakai-vue => exists
https://github.com/primefaces/sakai-nuxt => not found
